app/library/Utils.php

```diff
@@ -25,14 +25,13 @@ class Utils extends Component
      */
     public function fetch($data, $element, $default = '')
     {
-        $return = $default;
         if (true === is_object($data) && true === isset($data->$element)) {
-            $return = $data->$element;
+            return $data->$element;
         } elseif (true === is_array($data) && true === isset($data[$element])) {
-            $return = $data[$element];
+            return $data[$element];
         }
 
-        return $return;
+        return $default;
```

Comment:
Actually this was all done on purpose. There is the school of thought that you should always have one exit point or `return` statement in a method and this is what is shown here.

Indeed issuing a `return` exactly where it is needed (as you refactored it) is the fastest way possible for the function but in much larger methods all the `return` statements will cause maintenance headaches.